lock.go

```diff
@@ -3,17 +3,21 @@ package nuki
 import (
 	"context"
 	"fmt"
+	"github.com/tarent/go-nuki/communication"
 	"github.com/tarent/go-nuki/communication/command"
 )
 
-// ReadLockState will request the current lock state for the connected and paired nuki device and return the result.
-func (c *Client) ReadLockState(ctx context.Context) (command.KeyturnerStatesCommand, error) {
+// ReadLockerState will request the current locker state for the connected and paired nuki device and return the result.
+func (c *Client) ReadLockerState(ctx context.Context) (command.KeyturnerStatesCommand, error) {
```

Comment:
Mabe i should also generalize the state such like the config (for consistency)

Reply:
for the lock action we cant do that because there is a slightly different action id used for open (0x03) and unlocking (0x01)